charts/fission-all/values.yaml

```diff
@@ -155,6 +155,26 @@ executor:
     runAsUser: 10001
     runAsGroup: 10001
 
+  ## Object Reaper
+  ## objectReaperInterval (seconds) represents GLOBAL interval to run process that reaps objects after certain idle time.
+  ## Also you can set different objectReaperInterval for specific executor type. See poolmgs/newdeploy/container section
+  ## Default: 5
+  ##
+  objectReaperInterval: 5
+
+  poolmgr: {}
```

Comment:
instead of creating poolmgr, newdeploy and container as list type, we can create this as simple field name as poolmgrObjReaperInterval, newdeployObjReaperInterval, containerObjReaperInterval with default value empty.

poolmgrObjReaperInterval: ""
newdeployObjReaperInterval: ""
containerObjReaperInterval: ""

Reply:
This is for the future, most likely executors will have more variables.
For example `defaultIdlePodReapTime`. `useIstio` also good candidate to move from code to config files.